Oracle.md

```diff
@@ -0,0 +1,165 @@
+# Oracle specifications
+
```

Comment:
We will want a table of contents before merging

Reply:
aded